server/api/design/field_note.go

```diff
@@ -0,0 +1,139 @@
+package design
+
+import (
+	. "github.com/goadesign/goa/design"
+	. "github.com/goadesign/goa/design/apidsl"
+)
+
+var AddFieldNotePayload = Type("AddFieldNotePayload", func() {
```

Comment:
I actually think a better way is just to pull user_id for the currently authenticated user, so they actually don't need to provide this.